### DISCUSSION TITLE
Large memory clusters

### DISCUSSION THREAD

**Discussion by mflynn-lanl** [microbiomedata/nmdc-edge#305] [General]
@edwins
I created a large memory cluster (snapshot from a large memory VM) and used a large memory VM type for the worker node, but I didn't include a large enough volume to store the Cromwell executions data. So I created a new one with more scratch space but that one errored during creation. I created another cluster, this time using the same VM snapshot I used for our regular cluster, and a large-memory (r3xl) for the worker node. This cluster was created successfully, I could ssh to it and call `sinfo` but at some point later on when I tried to ssh in again, I was not able to because the ssh login timed out

**Comment by mflynn-lanl** [comment]
This is not an issue for clusters made with m3.2xl worker nodes

**Comment by mflynn-lanl** [comment]
It is working now. I can create a cluster with a r3.xl worker node. Not sure why it didn't work before using the same deployment parameters

**Comment by edwins** [comment]
one thing to consider is that when you create a snapshot, any future vms must be the size of the vm you snapshotted or larger. I think you might be hitting upon that issue.

A good practice is if you need to create a snapshot, launch a vm with the smallest size possible. that way, you will have flexibility in launching other sizes in the future.
